sim/build.sbt

```diff
@@ -59,7 +60,7 @@ lazy val midas = (project in file("midas"))
   )
 
 lazy val firesimLib = (project in file("firesim-lib"))
-  .settings(commonSettings).dependsOn(midas, icenet, testchipip, sifive_blocks)
+  .settings(commonSettings).dependsOn(midas, icenet, testchipip, chipyard, sifive_blocks)
```

Comment:
No cyclical dependency?

Reply:
`firechip` depends on `firesimLib`, but `chipyard` does not depend on `firechip`

Reply:
David's suggested change of moving `DefaultFiresimTestHarness` will remove dependency between `firesimLib` and `chipyard.